dubbo-filter/dubbo-filter-cache/src/main/java/org/apache/dubbo/cache/Cache.java

```diff
@@ -25,4 +25,10 @@
 
     Object get(Object key);
 
+    boolean remove(Object key);

```

Comment:
how to use it?

Reply:
when user want to Invalidate cache,espescially in ThreadLocalCache.details about this commit can be found in [#2592](https://github.com/apache/incubator-dubbo/issues/2592)